add support for custom tag in gitops template (#266)
Staffbase/gha-workflows: README.md

      docker-file: <path-to-Dockerfile>
      # optional: name of the docker image, default: private/<repository_name>
      docker-image: <your-image>

>       # optional: custom tag for the docker image. If set, will always be preferred from the tag generated by the workflow.

Reviewer: ```suggestion
      # optional: custom tag for the productive docker image which is preferred over the tag generated by the workflow
```
